### PR TITLE
Adding resume journey controller

### PR DESCRIPTION
**Pull request by sampankumar**
Adding resume journey controller

### CODE REVIEW

**Review by sardhani-ch — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by ttingle-ch — DISMISSED** [review]
LGTM

File: src/main/types/pageURL.ts [hunk 1]
```diff
@@ -6,6 +6,9 @@ export const START = "/"; // Domain name will go here
 
 export const HOME_URL = "";
 
+const TRANSACTION = "transaction";
+const TRANSACTION_PATH = TRANSACTION + "/:transactionId";
```
[the remark this answers is not in the document]
Comment (sampankumar) [reply src/main/types/pageURL.ts:10]: Addressed

**Review by swalke-ch — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**